dataactcore/config_example.yml

```diff
@@ -37,12 +37,12 @@ broker:
     # File D1 API
     # Uses "{}" because it requires string formatting to insert query parameters
     # If the URL is provided, the file name provided will be used when uploading to S3
-    award_procurement_url: https://data-usas.uat.workplace.gov/api/contracts?CGAC={}&date_signed_beginning={}&date_signed_end={}
+    award_procurement_url: https://data-usas.uat.workplace.gov/api/contracts?CGAC={}&date_signed_beginning={}&date_signed_end={}&callback={}
```

Comment:
See [earlier comment](https://github.com/fedspendingtransparency/data-act-broker-backend/pull/248/commits/237935d17d58fd54bee733496d14d72df4a8813d#r77886425) about where we should detail the parameter names.